Homework1: Tatiana Smatova

SmatovaTatiana opened the pull request:
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle).
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks.
- [x] my PR made from the fork feature branch to appropriate branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

SmatovaTatiana committed:
com 1

SmatovaTatiana committed:
del

SmatovaTatiana committed:
first commit

SmatovaTatiana committed:
first commit

SmatovaTatiana committed:
Add 6 tasks

SmatovaTatiana committed:
fix bugs

SmatovaTatiana committed:
Update HomeWork_1.py

Moved "Import" on the top

SmatovaTatiana committed:
Update HomeWork_1.py

SmatovaTatiana committed:
fixed

SmatovaTatiana committed:
Update HomeWork_1.py

fixed

belovmd commented on src/HomeWork_1.py:
```diff
@@ -0,0 +1,36 @@
+    print('This generation has {0} babies'.format(babies))
+    parents, babies = (babies, parents + babies)
+
+# 5 lines: Functions
```
2 lines between docstring and code

belovmd commented on src/HomeWork_1.py:
```diff
@@ -0,0 +1,36 @@
+name = input('What is your name?\n')
+print('Hi, %s.' % name)
+
+# 3 lines: For loop, built-in enumerate function, new style formatting
```
It has sense to do it like:
```
pass  # last line of previous task


# next task docstring
pass  # my next task first line  
```

belovmd commented on src/HomeWork_1.py:
```diff
@@ -0,0 +1,36 @@
+# 6 lines: Import, regular expressions
+import re
+for test_string in ['555-1212', 'ILLEGAL']:
```
should be at least 1 line between import and code

belovmd reviewed (changes requested):
you have to add additional code (80 lines for +1)

SmatovaTatiana committed:
Added some tasks more.
Formatted in following view: 1 line between docstring and code and 2 lines between last line of previous task and the docstring of the next task

SmatovaTatiana committed:
Merge remote-tracking branch 'origin/HomeWork_1' into HomeWork_1

# Conflicts:
#	src/HomeWork_1.py

SmatovaTatiana committed:
Added some tasks more.
Formatted in following view: 1 line between docstring and code and 2 lines between last line of previous task and the docstring of the next task

SmatovaTatiana committed:
try to fix

SmatovaTatiana committed:
try to fix

SmatovaTatiana committed:
try to fix

SmatovaTatiana committed:
try to fix_1

SmatovaTatiana committed:
try to fix one more time

belovmd reviewed (approved):
1
